### PR TITLE
Contributing conventions other improvements

### PR DESCRIPTION
**Pull request by jlevy**
Couple other small changes.

Also updated note on CloudFormation YAML, thanks to @benkehoe — review welcome.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -141,7 +152,7 @@ General Information
...
 -	⛓**Lock-in:** As you use AWS, it’s important to be aware when you are depending on AWS services that do not have equivalents elsewhere.
-	-	Lock-in may be completely fine for your company. But it’s important from a business perspective to make this choice explicitly, and consider the cost, operational, business continuity, and competitive risks of being tied to AWS. AWS is such a dominant and reliable vendor, many companies are comfortable with using AWS to its full extent. Other companies avoid this.
+	-	Lock-in may be completely fine for your company, or a significant risk. It’s important from a business perspective to make this choice explicitly, and consider the cost, operational, business continuity, and competitive risks of being tied to AWS. AWS is such a dominant and reliable vendor, many companies are comfortable with using AWS to its full extent. Others can tell stories about the [dangers “cloud jail” when costs spiral](http://firstround.com/review/the-three-infrastructure-mistakes-your-company-must-not-make/).
```
Comment (ThanosBaskous) [inline README.md]: "dangers of"

File: README.md [hunk 2]
```diff
@@ -247,24 +258,24 @@ Many services within AWS can at least be compared with Google Cloud offerings or
...
+| File storage                  | S3                                                                           | Cloud Storage                | GFS             |            |                                   | Swift, HDFS                                                |
 | Block storage                 | EBS                                                                          | Persistent Disk              |                 |            |                                   | NFS                                                        |
-| SQL datastore                 | RDS                                                                          | Cloud SQL                    |                 | SQL Database           |                                   | MySQL, PostgreSQL                                          |
+| SQL datastore                 | RDS                                                                          | Cloud SQL                    |                 |            |                                   | MySQL, PostgreSQL                                          |
```
Comment (ThanosBaskous) [inline README.md]: Intentionally removing?

File: README.md [hunk 3]
```diff
@@ -247,24 +258,24 @@ Many services within AWS can at least be compared with Google Cloud offerings or
...
 | Key-value store, column store | DynamoDB                                                                     | Cloud Datastore              | Megastore       |            |                                   | Cassandra, CouchDB, RethinkDB, Redis                       |
 | Memory cache                  | ElastiCache                                                                  | App Engine Memcache          |                 |            |                                   | Memcached, Redis                                           |
 | Search                        | CloudSearch, Elasticsearch (managed)                                         |                              |                 |            | Algolia, QBox                     | Elasticsearch, Solr                                        |
-| Data warehouse                | Redshift                                                                     | BigQuery                     |                 | SQL Data Warehouse           | Oracle, IBM, SAP, HP, many others | Greenplum                                                  |
```
Comment (ThanosBaskous) [inline README.md]: Ditto

File: README.md [hunk 4]
```diff
@@ -247,24 +258,24 @@ Many services within AWS can at least be compared with Google Cloud offerings or
...
-| Business intelligence         | QuickSight                                                                   |                              |                 | Power BI           | Tableau                           |                                                            |
```
Comment (ThanosBaskous) [inline README.md]: Ditto

File: README.md [hunk 5]
```diff
@@ -247,24 +258,24 @@ Many services within AWS can at least be compared with Google Cloud offerings or
...
 | Lock manager                  | [DynamoDB (weak)](https://gist.github.com/ryandotsmith/c95fd21fab91b0823328) |                              | Chubby          |            |                                   | ZooKeeper, Etcd, Consul                                    |
 | Message broker                | SQS, SNS, IoT                                                                | Pub/Sub                      | PubSub2         |            |                                   | RabbitMQ, Kafka, 0MQ                                       |
 | Streaming, distributed log    | Kinesis                                                                      | Dataflow                     | PubSub2         | Event Hubs |                                   | Kafka Streams, Apex, Flink, Spark Streaming, Storm         |
-| MapReduce                     | EMR                                                                          | Dataproc                     | MapReduce       | HDInsight           | Qubole                            | Hadoop                                                     |
```
Comment (ThanosBaskous) [inline README.md]: Ditto

File: README.md [hunk 6]
```diff
@@ -247,24 +258,24 @@ Many services within AWS can at least be compared with Google Cloud offerings or
 
 | Service                       | AWS                                                                          | Google Cloud                 | Google Internal | Microsoft  | Other providers                   | Open source “build your own”                               |
 |-------------------------------|------------------------------------------------------------------------------|------------------------------|-----------------|------------|-----------------------------------|------------------------------------------------------------|
-| Virtual server                | EC2                                                                          | Compute Engine (GCE)         |          | Azure VM      | DigitalOcean                      | OpenStack                                                  |
```
Comment (ThanosBaskous) [inline README.md]: Intentionally removing these?

Comment (jlevy) [inline README.md]: Yikes, bad merge.

**Pull request merged**